cassandane/Cassandane/Cyrus/Conversations.pm

```diff
@@ -260,32 +260,40 @@ sub test_move_200
 
     my $jmap = $self->{jmap};
     xlog $self, "create bar mailbox";
-    $res = $jmap->CallMethods([
+    $res = $jmap->CallMethods(
+        [
             ['Mailbox/set', { create => { "1" => {
                             name => "bar",
              }}}, "R1"]
-    ]);
+        ],
+        [ qw( urn:ietf:params:jmap:core urn:ietf:params:jmap:mail ) ],
```

Comment:
The issue is the test doesn't enable jmap_nonstandard_extensions, so 'https://cyrusimap.org/ns/jmap/mail' isn't allowed. The commit message isn't quite clear on that